### PR TITLE
🧪 Add story ad rendering to inabox.

### PR DESCRIPTION
**Pull request by calebcordry**
Introduces new logic to inabox that detects for story ad metadata, and if it exists, renders as a story ad. 

Part of #27851

### CODE REVIEW

File: src/inabox/inabox-story-ad.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * limitations under the License.
+ */
+
+import {ButtonTextFitter} from '../../extensions/amp-story-auto-ads/0.1/story-ad-button-text-fitter';
+import {
+  createCta,
+  getStoryAdMetadataFromDoc,
+  maybeCreateAttribution,
+  validateCtaMetadata,
+} from '../../extensions/amp-story-auto-ads/0.1/story-ad-ui';
```
Comment (calebcordry) [inline src/inabox/inabox-story-ad.js:23]: I know we discussed importing from extension being a little weird, but I wasn't sure where to put them. Happy to move to a new file path.

**Review by danielrozenberg — APPROVED** [review]
Approval for `build-system/`

File: src/inabox/inabox-story-ad.js (hunk 1 above)
Comment (lannka) [reply src/inabox/inabox-story-ad.js:23]: i'm fine with this as long as the compiler does not complain

**Review by lannka — APPROVED** [review]

File: src/inabox/inabox-story-ad.js (hunk 1 above)
Comment (rsimha) [reply src/inabox/inabox-story-ad.js:23]: @lannka @calebcordry Looks like this did break compilation. https://travis-ci.com/github/ampproject/amphtml/jobs/405408314#L418

Can we fix before this goes to beta tomorrow?

**Pull request merged**